docs/usermanual/userman_admin.rst

```diff
@@ -482,15 +481,15 @@ When you have copied in the files, to register them with KA Lite, follow the fol
 
 1. Click the "Manage" tab at the top of the page.
 2. Click on the "Videos" tab.
+3. Click the blue "Scan content folder for videos" button in the top left box of the page.
 
 .. screenshot::
     :user-role: admin
-    :url: /management/zone/
+    :url: /update/videos/
     :navigation-steps:
-    :focus: li.video
+    :focus: #scan-videos | Click this to search your video folder for previously downloaded videos!
```

Comment:
Maybe better:  *Click this to search your* **content** *folder for previously downloaded videos!*